api/include/vehicles/kia_niro.h

```diff
@@ -0,0 +1,460 @@
+/**
+ * @file kia_niro.h
+ * @brief Kia Niro specific macros.
+ *
+ */
+
+
+#ifndef _KIA_NIRO_PLATFORM_INFO_H_
+#define _KIA_NIRO_PLATFORM_INFO_H_
+
+
+#include <stdint.h>
+
+
+// ********************************************************************
+//
+// WARNING
+//
+// The values listed here are carefully tested to ensure that the vehicle's
+// components are not actuated outside of the range of what they can handle.
+// By changing any of these values you risk attempting to actuate outside of the
+// vehicle's valid range. This can cause damage to the hardware and/or a
+// vehicle fault. Clearing this fault state requires additional tools.
+//
+// It is NOT recommended to modify any of these values without expert knowledge.
+//
+// ************************************************************************
+
+
+// ****************************************************************************
+// OBD MESSAGES
+// ****************************************************************************
+
+/*
+ * @brief ID of the Kia Niro's OBD steering wheel angle CAN frame.
+ *
+ */
+#define KIA_SOUL_OBD_STEERING_WHEEL_ANGLE_CAN_ID ( 0x2B0 )
+
+/*
+ * @brief ID of the Kia Niro's OBD wheel speed CAN frame.
+ *
+ */
+#define KIA_SOUL_OBD_WHEEL_SPEED_CAN_ID ( 0x386 )
+
+/*
+ * @brief ID of the Kia Niro's OBD brake pressure CAN frame.
+ *
+ */
+#define KIA_SOUL_OBD_BRAKE_PRESSURE_CAN_ID ( 0x220 )
+
+/*
+ * @brief ID of the Kia Niro's OBD speed CAN frame.
+ *
+ */
+#define KIA_SOUL_OBD_SPEED_CAN_ID ( 0x371 )
+
+/*
+ * @brief Factor to scale OBD steering angle to degrees
+ *
+ */
+#define KIA_SOUL_OBD_STEERING_ANGLE_SCALAR ( 0.1 )
```

Comment:
The steering angle CAN signal is a signed integer that's expressed in 1/10th degrees, but the signness is reverse of what we expect - in that right is negative and left is positive. This is the reverse of what I'd expect - but changing this also means changing this for the other vehicles. Doing so a is an API breaking change so we should do this at a major version boundary.

We should ticket this defect and deal with it later.